### PR TITLE
Updating Community Guidelines Page

### PR DESCRIPTION
**Pull request by reyrodrigez**
Fixing issue https://github.com/HabitRPG/habitrpg/issues/5281

### CODE REVIEW

File: website/views/static/community-guidelines.jade [hunk 1]
```diff
@@ -30,14 +30,14 @@ block content
         |&nbsp;
         =env.t('February')
         |&nbsp;9&comma; 2015
-      h2=env.t('commGuideHeadingWelcome')
+      h2(id="welcome")=env.t('commGuideHeadingWelcome')
```
Comment (crookedneighbor) [inline website/views/static/community-guidelines.jade]: jade has a shorthand for this: `h2#welcome=env.t('commGuideHeadingWelcome')`

Would you mind making that change every place you've added ids?

File: website/views/static/community-guidelines.jade [hunk 2]
```diff
@@ -30,14 +30,14 @@ block content
...
       div(class='clearfix')
         img(class='pull-left', src='/community-guidelines-images/intro.png', alt='')
```
Comment (crookedneighbor) [inline website/views/static/community-guidelines.jade]: If you want to be super awesome, you could change the things that look like `element(class='class-name')` to `element.class-name`.

That's not strictly required for this PR, but would be nice to be consistent.

File: website/views/static/community-guidelines.jade [hunk 3]
```diff
@@ -81,6 +81,8 @@ block content
         img(class='pull-left', src='/community-guidelines-images/moderators.png', alt='')
         ul(class='pull-left list-unstyled')
           li
+            strong Beffymaroo
```
Comment (crookedneighbor) [inline website/views/static/community-guidelines.jade:84]: Not sure if this matters, but it's styled all lowercase: beffymaroo

@lemoness?

File: website/views/static/community-guidelines.jade [hunk 4]
```diff
@@ -30,14 +30,14 @@ block content
         |&nbsp;
         =env.t('February')
         |&nbsp;9&comma; 2015
```
Comment (crookedneighbor) [inline website/views/static/community-guidelines.jade]: The date should also be updated. If more than a day goes by before this is merged in, after you've adjusted it, I'll manually adjust it myself after it gets merged.

File: website/views/static/community-guidelines.jade (hunk 2 above)
Comment (reyrodrigez) [inline website/views/static/community-guidelines.jade]: Sure, will do, no worries.

File: website/views/static/community-guidelines.jade (hunk 3 above)
Comment (Alys) [inline website/views/static/community-guidelines.jade:84]: People's names should always be written exactly as the person themself has
chosen. For moderators, you can see their chosen names in the list at the
top of Tavern chat. So for beffymaroo, it must be written like that.

**Comment by crookedneighbor** [comment]
Merged! I made one change: https://github.com/HabitRPG/habitrpg/commit/0e2f0596f68cc32b27cae1d86f5831caec0471d3

In jade, if you don't specify an element, it assumes a `div`. 

This will count towards a contributor tier! What's your User ID? You can find it in [Settings > API](https://habitrpg.com/#/options/settings/api). Be sure to only copy the User ID and _not_ your API Token.   

Also, you may want to consider creating a separate git branch when submitting PRs. That will allow you to work on multiple issues at once! (and prevent changes for one issue from appearing in a PR for another one)

**Comment by reyrodrigez** [comment]
Nice! my id is `d35345f9-0b31-49b7-a4a7-ab7c062ed0a8`.
Thanks!

**Comment by crookedneighbor** [comment]
Congratulations, you've achieved your first contributor tier!

You can [read more about contributing here](http://habitrpg.wikia.com/wiki/Contributor_Rewards)

**Pull request merged**